src/content/docs/en/guides/on-demand-rendering.mdx

```diff
@@ -0,0 +1,258 @@
+---
+title: On-demand Rendering
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import { Steps } from '@astrojs/starlight/components';
+import RecipeLinks from '~/components/RecipeLinks.astro';
+import IntegrationsNav from '~/components/IntegrationsNav.astro';
+
+Astro allows you to choose **on-demand rendering** for some, or all of your pages and endpoints. 
+
+Generating HTML pages on the server when requested and sending them to the client is also known as **server-side rendering (SSR)**. An **adapter** is used to run your project on the server and handle these requests.
+
+## Official Adapters
+
+Astro maintains official adapters for [Node.js](https://nodejs.org/), [Vercel](https://vercel.com/), [Netlify](https://www.netlify.com/), and [Cloudflare](https://www.cloudflare.com/). 
+
+Find even more [community-maintained adapters](https://astro.build/integrations/?search=&categories%5B%5D=adapters) (e.g. Deno, SST, AWS) in our integrations directory. 
+
+<IntegrationsNav category="adapter"/>
+
+### Add an Adapter
+
+To render any of your pages on demand, for any `output` mode, you need to add an **adapter**.
+
+Each adapter allows Astro to output a script that runs your project on a specific *runtime*: the environment that runs code on the server to generate pages when they are requested (e.g. Netlify, Cloudflare).
+
+You can find both [official and community adapters in our integrations directory](https://astro.build/integrations/?search=&categories%5B%5D=adapters). Choose the one that corresponds to your [deployment environment](/en/guides/deploy/).
+
+You can add any of the [official adapters maintained by Astro](/en/guides/integrations-guide/#official-integrations) with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step. 
+
+For example, to install the Netlify adapter, run:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npx astro add netlify
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm astro add netlify
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro add net
```

Comment:
I think the name is the same for yarn:
```suggestion
  yarn astro add netlify
```

Reply:
hahaha, I actually stopped to tell someone I was updating these to Netlify, then forgot to come back and finish